model/src/wav_history_mod.F90

```diff
@@ -108,6 +108,9 @@ subroutine write_history ( timen )
     character(len=1024) :: fname
     character(len=12)   :: vname
     character(len=16)   :: user_timestring    !YYYY-MM-DD-SSSSS
+    ! indicator logfile
+    character(len=256)  :: log_fname          ! log file name
+    integer             :: log_unit = 28888   ! unit number for log file
```

Comment:
A better choice would be not hardwire a unit number, but to obtain a free one using ``open(newunit=log_unit....``.

Reply:
Issue opened to improve this in the next update: https://github.com/NOAA-EMC/WW3/issues/1400